Sources/TensorFlow/Optimizers/MomentumBased.swift

```diff
@@ -207,13 +207,12 @@ where
 /// (Kingma and Ba, 2014).
 ///
 /// - Parameters:
```

Comment:
Sorry I overlooked this: this entire doc comment belongs on the `Adam.init` declaration, not on the `Adam` type declaration itself. Would you mind moving it there?

Reply:
@dan-zheng like so?

Reply:
Not quite - doc comments always belong above declarations, not in their bodies.
```
  /// - Parameters:
  /// ...
  public init(
    for model: __shared Model,
    learningRate: Float = 1e-3,
    beta1: Float = 0.9,
    beta2: Float = 0.999,
    epsilon: Float = 1e-8,
    decay: Float = 0
  ) {
```

Be sure to use `/// - Parameters:` with a leading hyphen. https://swift.org/documentation/api-design-guidelines/#write-doc-comment

Reply:
@dan-zheng moved again, re-added the leading hyphen!